cache-config/testing/ort-tests/t3c-lockfile_test.go

```diff
@@ -76,38 +74,36 @@ func TestLockfile(t *testing.T) {
 		}
 
 		if acquireStartLine == "" || acquireEndLine == "" {
-			t.Fatalf("t3c apply output expected to contain 'Trying to acquire app lock' and 'Acquired app lock', actual: '''%s'''", out)
+			t.Fatalf("t3c apply output expected to contain 'Trying to acquire app lock' and 'Acquired app lock', actual: %s", out)
 		}
 
-		acquireStart := parseLogLineTime(acquireStartLine)
-		if acquireStart == nil {
-			t.Fatalf("t3c apply acquire line failed to parse time, line '" + acquireStartLine + "'")
+		acquireStart, err := parseLogLineTime(acquireStartLine)
+		if acquireStart == nil || err != nil {
+			t.Fatalf("t3c apply acquire line failed to parse time, line '%s': %v", acquireStartLine, err)
 		}
-		acquireEnd := parseLogLineTime(acquireEndLine)
-		if acquireEnd == nil {
-			t.Fatalf("t3c apply acquire line failed to parse time, line '" + acquireEndLine + "'")
+		acquireEnd, err := parseLogLineTime(acquireEndLine)
+		if acquireEnd == nil || err != nil {
+			t.Fatalf("t3c apply acquire line failed to parse time, line '%s': %v", acquireEndLine, err)
 		}
 
 		minDiff := time.Second * 1 // checking the file lock should never take 1s, so that's enough to verify it was hit
 		if diff := acquireEnd.Sub(*acquireStart); diff < minDiff {
 			t.Fatalf("t3c apply expected time to acquire while another t3c is running to be at least %s, actual %s start line '%s' end line '%s'", minDiff, diff, acquireStartLine, acquireEndLine)
 		}
 
-		t.Logf(testName + " succeeded")
 	})
-	t.Logf("------------- End of " + testName + " ---------------")
 }
 
-func parseLogLineTime(line string) *time.Time {
+func parseLogLineTime(line string) (*time.Time, error) {
```

Comment:
Not a huge deal, but I intentionally didn't add detailed errors to this, because this test isn't the right place to test the log format

Reply:
I just think it helps with debugging to know why the parse failed, if it does. I can revert it, though, the odds of that happening are probably pretty low, and/or in that case some more specific test is probably failing.